packages/backend/src/payment-method/ilp/connector/core/middleware/balance.ts

```diff
@@ -75,15 +75,13 @@ export function createBalanceMiddleware(): ILPMiddleware {
 
     if (state.streamDestination) {
       await next()
+      if (response.fulfill) await createTransfer()
```

Comment:
can we have a test for the single phase transfer? (we can at least probably assert that `services.accounting.createTransfer` was called without the timeout set)